### PR TITLE
📦 🛢 [hashids] Use Hash IDs with every public-facing ID

### PR DESCRIPTION
**Pull request by wlonk**
baby gecko

### CODE REVIEW

File: metadeploy/api/models.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 from asgiref.sync import async_to_sync
 from colorfield.fields import ColorField
 from model_utils import Choices, FieldTracker
+from hashid_field import HashidAutoField
```
Comment (cdcarter) [inline metadeploy/api/models.py]: oh hey I didn't know about this django implementation, awesome!

Comment (wlonk) [reply metadeploy/api/models.py]: It's pretty good; there's a little mishegas to get it to play well with DRF, but not too bad.

File: env.example [hunk 2]
```diff
@@ -11,6 +11,7 @@ export POSTGRES_DB=metadeploy
 # General settings
 export DJANGO_SETTINGS_MODULE=config.settings.local
 export DJANGO_SECRET_KEY='CHANGEME!!!'
+export DJANGO_HASHID_SALT='something long and random'
```
Comment (jgerigmeyer) [inline env.example]: If/when this gets backported to project-template, we'll have to update the post-hook to replace this string.

Comment (wlonk) [reply env.example]: I already changed the `app.json`, but didn't think about the cookiecutter post-hook part.

Comment (jgerigmeyer) [reply env.example]: I went ahead and made the project-template changes locally; I'll push when we merge this branch and I backport it.

**Pull request merged**